command/cp.go

```diff
@@ -382,6 +388,7 @@ func (c Copy) doUpload(ctx context.Context, srcurl *url.URL, dsturl *url.URL) er
 	metadata := map[string]string{
```

Comment:
```go
metadata := storage.Metadata {
      "StorageClass": string(c.storageClass),
      "ContentType":  guessContentType(f),
      "ACL":          c.acl,
}
```

Same applies to [line 429](https://github.com/peak/s5cmd/blob/7da33dcd2367e3639bb263ec26c5b617c09febf2/command/cp.go#L429)